src/Network/Response.php

```diff
@@ -419,6 +436,16 @@ public function __construct(array $options = [])
         if (isset($options['body'])) {
             $this->body($options['body']);
         }
+
+        if (isset($options['input'])) {
```

Comment:
Isn't this the same as `body`? Responses don't really have an input, just a body.